src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-local_ja.html

```diff
@@ -1,5 +1,5 @@
-<div>
-  このモジュールをチェックアウトするローカルのディレクトリ(<a href="ws" target="_blank">ワークスペースのルート</a>からの相対パス)
-  を指定してください。
-  何も設定しないと、コマンドの<tt>svn</tt>と同様に、URLの最後のパス名をデフォルトとして使用します。
-</div>
+<div>
```

Comment:
Git converted line endings from windows style to unix style. 
All other files appear to use unix style line endings.